core/bridges/orm.go

```diff
@@ -165,6 +170,28 @@ func (o *orm) UpdateBridgeType(bt *BridgeType, btr *BridgeTypeRequest) error {
 	return err
 }
 
+func (o *orm) GetLastGoodResponse(dotId string, specId int32, maxElapsed time.Duration) (response []byte, err error) {
+	stalenessThreshold := time.Now().Add(-maxElapsed)
+	sql := `SELECT value FROM bridge_last_value WHERE
+				dot_id = $1 AND 
+				spec_id = $2 AND 
+				finished_at > ($3)	
```

Comment:
Maybe @reductionista would confirm here. If only `finished_at` is indexed, does it make sense to make this condition appearing first in all `WHERE`?

Reply:
afaik postgres does not care about where clause order